.github/pull_request_template.md

```
@@ -0,0 +1,342 @@
+![alt text](https://upload.wikimedia.org/wikipedia/commons/thumb/2/27/Farmatodo_logo.svg/1280px-Farmatodo_logo.svg.png)
+# Revisión de Código
+
+### Design docs asociados -> [Design doc feature](#https://www.farmatodo.com.co)
```

Markdownlint

Expected: h2; Actual: h3
Heading levels should only increment by one level at a time

(MD001, heading-increment)